lab2.md

```diff
@@ -4,9 +4,7 @@
 
 ## Use the UI Portal
 
-You can now test out the OpenFaaS UI by going to http://localhost:8080 or http://127.0.0.1:8080 - if you're deploying to a Linux VM then replace localhost with the IP address from the output you see on the `ifconfig` command.
-
-Note: on some Linux distributions accessing `localhost` may hang, if that happens then please use `127.0.0.1` instead and replace it wherever you see `localhost`.
```

Comment:
> Note: We could say that we're using 127.0.0.1 instead of `localhost` which may hang on some Linux systems due to conflicts between IPv4/IPv6 networking.